recipes/libbezier/bld.bat

```diff
@@ -0,0 +1,49 @@
+:: NOTE: This assumes the following environment variables have been set.
+::       - `%SRC_DIR%`
+::       - `%LIBRARY_PREFIX%`
+:: H/T: https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html
+
+mkdir build
+cd build
+
+:: NOTE: This "should be" the following
+::
+:: IF "%APPVEYOR%"=="True" (
+::     IF "%ARCH%"=="32" (
+::         set PATH=%PATH%;C:\mingw-w64\i686-6.3.0-posix-dwarf-rt_v5-rev1\mingw32\bin
+::     ) ELSE (
+::         set PATH=%PATH%;C:\mingw-w64\x86_64-8.1.0-posix-seh-rt_v6-rev0\mingw64\bin
+::     )
+:: )
+::
+:: OR
+::
+:: set PATH=%PATH%;C:\mingw-w64\x86_64-8.1.0-posix-seh-rt_v6-rev0\mingw64\bin;C:\mingw-w64\i686-6.3.0-posix-dwarf-rt_v5-rev1\mingw32\bin
+::
+:: but the pre-processing of `bld.bat` causes issues with the indents.
+set PATH=%PATH%;C:\mingw-w64\x86_64-8.1.0-posix-seh-rt_v6-rev0\mingw64\bin
+set PATH=%PATH%;C:\mingw-w64\i686-6.3.0-posix-dwarf-rt_v5-rev1\mingw32\bin
```

Comment:
I'm trying to get a non-failing run in AppVeyor (`gfortran` isn't on `%PATH%` by default), but the build fails at the outset with a Python 2.7 "oh no TLS!" [error][1]:

```
ff_ci_pr_build -v --ci "appveyor" "%APPVEYOR_ACCOUNT_NAME%/%APPVEYOR_PROJECT_SLUG%" "%APPVEYOR_BUILD_NUMBER%" "%APPVEYOR_PULL_REQUEST_NUMBER%"
Checking to see if this PR build is outdated.
Traceback (most recent call last):
  File "C:\projects\staged-recipes\ff_ci_pr_build.py", line 201, in <module>
    sys.exit(main())
  File "C:\projects\staged-recipes\ff_ci_pr_build.py", line 192, in main
    exit_code = int(appveyor_check_latest_pr_build(repo, pr, bld) is False)
  File "C:\projects\staged-recipes\ff_ci_pr_build.py", line 117, in appveyor_check_latest_pr_build
    data = request_json(url.format(repo=repo, total_builds=total_builds), headers=headers)
  File "C:\projects\staged-recipes\ff_ci_pr_build.py", line 46, in request_json
    with contextlib.closing(urlopen(request)) as response:
  File "C:\Python27\lib\urllib2.py", line 154, in urlopen
    return opener.open(url, data, timeout)
  File "C:\Python27\lib\urllib2.py", line 429, in open
    response = self._open(req, data)
  File "C:\Python27\lib\urllib2.py", line 447, in _open
    '_open', req)
  File "C:\Python27\lib\urllib2.py", line 407, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\urllib2.py", line 1241, in https_open
    context=self._context)
  File "C:\Python27\lib\urllib2.py", line 1198, in do_open
    raise URLError(err)
urllib2.URLError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:727)>
Command exited with code 1
```

[1]: https://ci.appveyor.com/project/conda-forge/staged-recipes/builds/30683289